detection-rules/homoglyph_sender_domain_template.yml

```diff
@@ -5,10 +5,10 @@ type: "rule"
 source: |
   type.inbound
   and (
-      iedit_distance(sender.email.domain.domain, 'YOURDOMAIN1.COM') < 2
-      or iedit_distance(sender.email.domain.domain, 'YOURDOMAIN2.COM') < 2
+      strings.ilevenshtein(sender.email.domain.domain, 'YOURDOMAIN1.COM') < 2
```

Comment:
can we make this dynamic now? so replace lines 8 and 9 with:

`any($org_domains, string.ilevenshtein(sender.email.domain.domain, .) > 2)`

Reply:
hmm... technically we've had that functionality for a long time, so it's not new with these changes.
I'm a little hesitant because levenshtein distance can be slow, and if this list happens to be large then it could be a problem.

let's come back to this one after this PR since it's separate

Reply:
this file is meant to be deleted anyway: https://github.com/sublime-security/sublime-rules/pull/177

Reply:
and we have a dynamic version already: https://github.com/sublime-security/sublime-rules/blob/eb03d616e775eefa51417991fa57ac8dcaee25bd/detection-rules/lookalike_sender_domain.yml

Reply:
nice, separately let's consider adding "homoglyph" to the name and/or tags for that rule. we'll merge #177 shortly